apps/explorer/lib/explorer/chain.ex

```diff
@@ -5109,6 +5109,109 @@ defmodule Explorer.Chain do
     end
   end
 
+  def fetch_latest_delegate_call_txn(address_hash) do
```

Comment:
this method fetch query an address last 10 success txn and pass result to fetch_internal_transaction_for_delegate_call